embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/api/EmbraceApiService.kt

```diff
@@ -145,29 +144,13 @@ internal class EmbraceApiService(
         post(eventMessage, mapper::eventMessageRequest)
     }
 
-    /**
-     * Sends an event to the API and waits for the request to be completed
-     *
-     * @param eventMessage the event message containing the event
-     */
-    override fun sendEventAndWait(eventMessage: EventMessage) {
-        post(eventMessage, mapper::eventMessageRequest).get()
-    }
-
     /**
      * Sends a crash event to the API and reschedules it if the request times out
      *
      * @param crash the event message containing the crash
      */
-    override fun sendCrash(crash: EventMessage) {
-        try {
-            post(crash, mapper::eventMessageRequest) { cacheManager.deleteCrash() }.get(
-                CRASH_TIMEOUT,
-                TimeUnit.SECONDS
-            )
-        } catch (e: Exception) {
-            logger.logError("The crash report request has timed out.")
-        }
+    override fun sendCrash(crash: EventMessage): Future<*> {
+        return post(crash, mapper::eventMessageRequest) { cacheManager.deleteCrash() }
```

Comment:
Could this increase the likelihood of the application process being terminated before the crash is deleted since we're not (needlessly) blocking the thread?